src/TableV2.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+  export let data = [];
+  export let highlight;
+  export let classes = '';
+</script>
+
+<div class="bg-[#efebc0] px-8 {classes}">
+  <table class="w-full table">
+    <tr class="header-row">
+      <slot name="headerColumns"/>
+    </tr>
+    {#each data as row, index}
+      <tr class:active={index === highlight} class="">
+        <slot name="columns" dataAtColumn={row} {index}/>
+      </tr>
+    {/each}
+  </table>
+</div>
+
+<style global>
+
+  .header-row:first-child th{
+    padding-top: 24px;
```

Comment:
can we transform those into tailwind classes? and if they don't match the tailwind sizes, adjust so they match tailwind (not the figma, the figma is for general guidance and look). thanks.